x-pack/plugins/cases/public/components/cases_context/cases_global_components.tsx

```diff
@@ -15,6 +18,9 @@ export const CasesGlobalComponents = React.memo(({ state }: { state: CasesContex
       {state.createCaseFlyout.isFlyoutOpen && state.createCaseFlyout.props !== undefined
         ? getCreateCaseFlyoutLazyNoProvider(state.createCaseFlyout.props)
         : null}
+      {state.selectCaseModal.isModalOpen && state.selectCaseModal.props !== undefined
```

Comment:
This is what effectively (lazy) renders the selection modal inside the cases context.